core/src/test/java/org/lfenergy/shapeshifter/core/service/validation/message/FlexOfferCurrencyValidatorTest.java

```diff
@@ -20,7 +20,7 @@
 
 class FlexOfferCurrencyValidatorTest {
 
-    private static final UftpParticipant DSO = UftpParticipantFixture.createTestAGRParticipant();
+    private static final UftpParticipant DSO = UftpParticipantFixture.AGR;
```

Comment:
Name of the constant is inconsistent with its actual value (DSO vs AGR). Apparently this was already wrong. Boyscout?
I'd also say you could remove the constant entirely and just import and refer to UftpParticipantFixture.AGR directly.

The same comment for the other validator test files.